applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -35,7 +35,8 @@ LOG_MODULE_REGISTER(cis_gateway, CONFIG_LOG_BLE_LEVEL);
 				  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU), 8, NULL);
 #define NET_BUF_POOL_PTR_ITERATE(i, ...) IDENTITY(&iso_tx_pool_##i)
 LISTIFY(CONFIG_BT_ISO_MAX_CHAN, NET_BUF_POOL_ITERATE, (;))
-
+#define DEVICE_NAME_PEER CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
```

Comment:
I think we should still have this in le_audio.h. Now we have the same define in both cis_headset and cis_gateway